javascript/javascript_sdk/src/sensr_client.js

```diff
@@ -0,0 +1,56 @@
+const WebSocket = require('ws');
+const outputMsg = require('./../js_proto/sensr_proto/output_pb.js');
+const pointMsg = require('./../js_proto/sensr_proto/point_cloud_pb');
+
+const SensrClient = endpoint => {
+    const socket_result = new WebSocket(`ws://${endpoint}:5050`);
+    socket_result.binaryType = 'arraybuffer';
+    const socket_point = new WebSocket(`ws://${endpoint}:5051`);
+    socket_point.binaryType = 'arraybuffer';
```

Comment:
Can I suggest renaming to `result_socket` and `point_socket`? Just my personal preference, feel free to ignore if you disagree